RUFAS/routines/manure/gas_emissions/calculator.py

```diff
@@ -448,13 +448,13 @@ def calculate_housing_ammonia_emission(
         return max(0.0, total_ammonia_loss)
 
     @classmethod
-    def storage_ammonia_emission(
+    def calculate_storage_ammonia_emission(
```

Comment:
```suggestion
    def calculate_liquid_storage_ammonia_emission(
```
If we're already changing the name, specifying the function is applicable to liquid storages is helpful because both liquid and solid manure storages calculate ammonia emissions, but call different methods in `calculator.py`. If you're just looking at calc.py, it can be confusing what is for what.